docs/DeveloperGuide.md

```diff
@@ -2,37 +2,159 @@
+details. If there is no text file, a text file will be created.
+
+### [DONE] List companies feature
+#### Implementation
+The proposed <code>CompanyList.printCompanyInformation</code> is facilitated by <code>ListCompanyCommand</code>. It will
+print out the companies that are stored in an ArrayList called companyList. The Company object is then printed out with 
+an integer denoting its index, four parameters, <code>String companyName </code>, <code>String industry </code>, 
+<code>Int contactNumber </code>, and <code>String contactEmail </code>, and its confirmation status. It will print out 
+each company from the entire companyList from index 0 to the final index of the last company stored in companyList.
+
+### [DONE] Store Company information to text file feature
+#### Implementation
+The proposed <code>CompanyListEncoder.write()</code> feature checks if a text file of name "companyList.txt" exists.
+If the text file does not exist, it is created when the programme boots up.
+If the text file exists, this method iterates through the list of company information in the list of companies and 
+stores the companies' information in the text file. Prior to writing the company information to this text file, all 
+previously saved text in this file is overwritten.
+![Encoder.png](UML%2FImage%2FEncoder.png)
+
+### [DONE] Load Company information from text file feature
+#### Implementation
+The proposed <code>CompanyListDecoder.load()</code> feature checks if a text file of name "companyList.txt" exists.
+If the text file does not exist, a message is printed to the user indicating that the file is not found.
+If the text file exists, this method iterates through the text file line by line and parses the input. Then the parsed
+input is added to the list of companies using the method <code>CompanyList.add</code>.
 
 ## Product scope
 ### Target user profile
 
-{Describe the target user profile}
+EveNtUS targets career fair event managers who are looking to better manage the companies attending the career fair, as well as managing the venue and crowd size. 
 
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+EveNtUS is a desktop application designed for career fair managers to manage career fairs, with a focus on efficient operation through the Command-Line Interface(CLI).
+
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ...   | I want to ...                                                                                                                                                        | So that I can ...                                                                        |
```

Comment:
Your user stories are well-structured and consistent. Good job. However, you might want to quantify what a basic user is.